nvflare/edge/executors/edge_model_executor.py

```diff
@@ -124,7 +123,33 @@ def _make_cookie(model_version, device_selection_id):
             CookieKey.DEVICE_SELECTION_ID: device_selection_id,
         }
 
-    def handle_task_request(self, request: TaskRequest, current_task: TaskInfo, fl_ctx: FLContext) -> TaskResponse:
+    def _handle_selection_request(
```

Comment:
don't quite get the idea of selection_request on how it functions beyond task_request

Reply:
Good question. This is to make simulator capable of supporting device reuse by the Assessor. When the Assessor reuses an old device, we need to make sure that the old device still has a chance to be selected by the Simulator.
But if the simulator only makes queries for the active devices (which won't include the old device any more), then the old device will never get a chance to be selected.

I solve this problem by always asking for the current selected devices at the beginning of each cycle, and then treat such devices with highest priority. This guarantees that the simulator will always send requests for selected devices.